updateHostsFile.py

```diff
@@ -21,17 +21,20 @@
 import time
 from glob import glob
 
-import lxml  # noqa: F401
-from bs4 import BeautifulSoup
-
 # Detecting Python 3 for version-dependent implementations
 PY3 = sys.version_info >= (3, 0)
 
-if PY3:
-    from urllib.request import urlopen
-else:
+if not PY3:
     raise Exception("We do not support Python 2 anymore.")
 
+
+try:
+    import requests
+except ModuleNotFoundError:  # noqa: F821
+    raise ModuleNotFoundError("This project's dependencies have changed. The Requests library ("  # noqa: F821
+                              "https://requests.readthedocs.io/en/master/) is now required.")
```

Comment:
If this error class was introduced in 3.6, then we should `except` on `ImportError` (`ModuleNotFoundError` is a sub-class) and raise that instead of `ModuleNotFoundError`.  Otherwise, 3.5 will break with a `NameError`.

Reply:
Indeed, it was behaving strangely locally so I thought it wouldn't be an issue. Would it be better to raise different errors based on whether the version is > 3.5, or just use an `ImportError` regardless?

Reply:
Keep `ImportError` regardless.  This doesn't need to be super elaborate (KISS).

Reply:
I don't think it would work anyway, so nvm. It all works out in the end!